### DISCUSSION TITLE
HTTP와 HTTPS에 대해서 설명하고 차이점에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by SionBackEnd** [Team-Psyduck/tech-sharing#73] [General]
오늘의 알고리즘 
[백준 부분합](https://www.acmicpc.net/problem/1806)

**Comment by eatingbug** [comment]
- HTTP는 Hpyertext transfer protocol 의 약자로 클라이언트와 서버간에 hypertext 를 주고받기위한 통신규약입니다.
- HTTPS는 HTTP를 SSL 및 TLS 기술에 결합하여 브라우저에서 전송된 정보를 제3자가 가로채서 읽지못하도록 암호화한 버전입니다.

**Comment by SionBackEnd** [comment]
HTTP란 Hypertext Transfer Protocol의 약자로 클라이언트와 서버 간 통신을 위한 통신 규칙(프로토콜)이다.
HTTP는 HTTP 메서드를 통해 요청하고 응답 코드를 통해 응답을 전송한다. 
HTTPS란 Hypertext Transfer Protocol Secure의 약자로 HTTP에서 보안이 추가된 프로토콜이다. 
SSL 인증서를 통해 사이트의 신뢰성을 검증한다. 

HTTP보다 HTTPS가 더 좋은 이유는 3가지가 존재한다. 
1. 보안
2. 검색 우선순위
3. 사이트 로드 속도

[참조](https://aws.amazon.com/ko/compare/the-difference-between-https-and-http/)
